crypto/threads_win.c

```diff
@@ -168,17 +184,23 @@ void ossl_rcu_lock_free(CRYPTO_RCU_LOCK *lock)
     OPENSSL_free(lock);
 }
 
-static inline struct rcu_qp *get_hold_current_qp(CRYPTO_RCU_LOCK *lock)
+static INLINE_VS struct rcu_qp *get_hold_current_qp(CRYPTO_RCU_LOCK *lock)
```

Comment:
We have `ossl_inline` please use that.

Reply:
Thanks! I changed it. Test build successful on Windows 2003, VS2010.